rosdep/python.yaml

```
@@ -9904,6 +9904,10 @@ yapf3:
   nixos: [python3Packages.yapf]
   ubuntu:
     '*': [yapf3]
+yoctopuce-pip:
```

Our current best practice is to put `python3-` in front of the key:

```suggestion
python3-yoctopuce-pip:
```

(don't forget to re-alphabetize this once you do that)

---

I have renamed it to `python3-yoctopuce` and placed it to the correct order alphabetically. Do you think it's better to add it at `base.yaml` ?

---

> I have renamed it to `python3-yoctopuce` and placed it to the correct order alphabetically. Do you think it's better to add it at `base.yaml` ?

No, it should be in `python.yaml`, but it should be called `python3-yocotopuce-pip`.

---

Done